support-frontend/assets/pages/paper-subscription-checkout/components/deliveryAgentsSelect.tsx

```diff
@@ -23,12 +23,15 @@ interface DeliveryAgentsSelectProps {
 		'addressMeta/setDeliveryAgent'
 	>;
 	formErrors: Array<FormError<string>>;
+	deliveryAddressErrors: Array<FormError<string>>;
 }
 
 export function DeliveryAgentsSelect(
 	props: DeliveryAgentsSelectProps,
 ): JSX.Element | null {
-	if (props.deliveryAgentsResponse?.type === 'Covered') {
+	const postcodeError = firstError('postCode', props.deliveryAddressErrors);
+
+	if (props.deliveryAgentsResponse?.type === 'Covered' && !postcodeError) {
```

Comment:
it didn't really make sense to have a postcode error and also show the delivery partner eg prevent this:
![image](https://github.com/guardian/support-frontend/assets/114918544/7d8dd2c0-29e1-4af4-87ce-5a3e8f26df9b)